linked_list/linked_list.py

```diff
@@ -0,0 +1,48 @@
+from linked_list.node import Node
+
+
+class LinkedList(object):
+    def __init__(self, head=None):
+        self.head = head
+
+    def insert(self, data):
+        new_node = Node(data)
+        new_node.set_next(self.head)
+        self.head = new_node
+
+    def size(self):
+        current = self.head
+        count = 0
+        while current:
+            count += 1
+            current = current.get_next()
+        return count
+
+    def search(self, data):
```

Comment:
the name of this method is the same but not the parameters, try to standardize between your two lists